torch/csrc/jit/fusion_compiler.h

```diff
@@ -46,7 +46,8 @@ struct TensorDesc {
 
   // do we have inner stride == 1?
   bool lastIsContiguous() const {
-    return contiguity.size() == 0 || contiguity.back();
+    // NB: A scalar tensor does not have a "last dimension" because it has 0 dims.
+    return contiguity.size() != 0 && contiguity.back();
```

Comment:
The code previously would still correctly handle the scalar case but would return `true`, which seems to makes sense. Everything is always contiguous if you have one element.